Bottom: 370d2e9d02288bf5f125beaf02726dc246efc1c4
Top:    f0a3fe692ffa4727c0e41e6ecff305a516a9633a
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-01-11 14:12:26 +0800

tools: fix virtio test build failure

Signed-off-by: Jason Wang <jasowang@redhat.com>


---

diff --git a/tools/include/xen/xen.h b/tools/include/xen/xen.h
new file mode 100644
index 0000000..44f993c
--- /dev/null
+++ b/tools/include/xen/xen.h
@@ -0,0 +1,6 @@
+#ifndef _XEN_XEN_H
+#define _XEN_XEN_H
+
+#define xen_domain()		(0)
+
+#endif	/* _XEN_XEN_H */
